packages/client/src/forms/configuration/default.ts

```diff
@@ -7706,6 +7704,14 @@ export const registerForms: IDefaultRegisterForms = {
         groups: [
           {
             id: 'documents-view-group',
+            conditionals: [
+              {
+                description: 'Hidden for record correction',
+                action: 'hide',
+                expression:
+                  'draftData && draftData.corrector && draftData.corrector.relationship'
+              }
+            ],
```

Comment:
The relationship with the corrector needs to be filled in when starting a record correction so in that case it can be considered a record correction and we can use that information to hide previous documents' section.